Essentials/src/main/java/com/earth2me/essentials/CommandSource.java

```diff
@@ -21,6 +27,46 @@ public final Player getPlayer() {
         return null;
     }
 
+    public void sendTl(final IEssentials ess, final String tlKey, final Object... args) {
+        if (isPlayer()) {
+            //noinspection ConstantConditions
+            getUser(ess).sendTl(tlKey, args);
+            return;
+        }
+
+        final String translation = tlLiteral(tlKey, args);
+        if (!translation.startsWith("MM||")) {
```

Comment:
Should shove `MM||` into a constant, this isn't the only line it is referenced/checked.